index.html

```diff
@@ -0,0 +1,506 @@
+            <a href="#"
+              class="text-white bg-primary_blue hover:bg-black hover:text-white px-5 py-1 rounded-3xl font-semibold w-fit">Start
+              now</a>
+            <a href="#" class="text-primary_blue font-semibold hover:text-black cursor-pointer">Contact sales</a>
+          </div>
+        </div>
+
+        <!-- <div class="mt-4 flex flex-col gap-8"> -->
+        <div class="flex flex-col gap-3 mt-4 md:mt-0 md:ml-10">
+          <img src="./assets/images/gear.png" class="w-12" alt="tools image">
+          <h4 class="font-semibold">Fastest-improving platform</h4>
+          <p>Integrated per-transaction pricing with no hidden fees</p>
+          <a href="#" class="text-primary_blue font-semibold hover:text-black cursor-pointer">Pricing details</a>
+        </div>
+        <div class="flex flex-col gap-3 mt-4 md:-mt-2 md:">
+          <img src="./assets/images/gear.png" class="w-12" alt="tools image">
+          <h4 class="font-semibold">Start your integration</h4>
+          <p>
+            Get up and running with Stripe in as little as 10 minutes.</p>
+          <a href="#" class="text-primary_blue font-semibold hover:text-black cursor-pointer">Pricing details</a>
+        </div>
+        <!-- </div> -->
+      </div>
+    </section>
+
+
+
+
+
+
+
+    <!--footer-->
+
+    <section class="mt-10 relative">
+      <div class="-skew-y-6  w-full bg-slate-100  h-[13vw] absolute z-10 top-0">
+        <div class="flex justify-between -mt-28  relative z-40">
+          <!-- item 1 -->
+          <div class="hidden lg:block -mt-24">
+            <div class="relative bg-cyan-400 -skew-y-6 w-52 h-10  ml-10  mt-40"></div>
+            <div class="bg-primary_blue   opacity-80 -skew-y-6 w-80 h-12  -mt-6 "></div>
+          </div>
+          <!-- item 2 -->
+          <div class="hidden lg:block mt-32">
+            <div class="relative bg-cyan-400 -mt-16 -skew-y-6 w-80 h-10    "></div>
+            <div class="bg-primary_blue relative  opacity-80 -skew-y-6 w-52 h-10 ml-10  -mt-4"></div>
+          </div>
+        </div>
+      </div>
+      <div class="bg-slate-100 relative top-[7vw] bg-no-repeat bg-center bg-cover md:px-28px">
+        <div class="md:px-32 px-8 py-20">
+          <div class="mt-8 grid grid-cols-2 gap-12 md:grid-cols-4 md:gap-4">
+
+            <!--1-->
+            <div class="flex flex-col gap-2 col-span-2 md:col-span-1 md:justify-between">
+              <div>
+                <img class="w-16 cursor-pointer " src="./assets/images/logo_white.svg" alt="">
+                <div class="flex items-center gap-2 md:mt-2">
+                  <svg class="LocaleControl__triggerIcon" width="16" height="16" viewBox="0 0 16 16">
+                    <path
+                      d="M3.54 8.04h4.42v4.43c0 .57.76.75 1.01.24l4.96-9.93a.54.54 0 0 0-.72-.72L3.3 7.03c-.5.25-.33 1.01.24 1.01"
+                      fill="#0A2540" fill-rule="evenodd"></path>
+                  </svg>
+                  <h4>United states</h4>
```

Comment:
Use the SVG images as files instead of directly using them in codes, It will help you to have fewer and readable lines of code